config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml

```diff
@@ -13,7 +13,7 @@ presubmits:
         - -c
         - |
           gcloud auth activate-service-account --key-file=/etc/gcp/service-account.json \
-          && gcloud container clusters get-credentials testing --zone=us-central1-f --project=rules-k8s \
+          && gcloud container clusters get-credentials testing --zone=us-west1-b --project=rules-k8s \
```

Comment:
I think this also need to stay in central1-f since it's an existing cluster...

Reply:
The only thing we should change is where we create new VMs. Existing resources (like rules-k8s' testing cluster in us-central1-f) are not effected